src/components/Transfer.tsx

```diff
@@ -60,7 +60,7 @@ const Transfer = () => {
   });
 
   const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    setTransferInput(event.target.value);
+    event.target.value.length <= 12 && setTransferInput(event.target.value);
```

Comment:
This prevents sending anything greater than 1000 tokens.

Also it is imho pretty bad experience when the input field is limited (like it enforces you to do changes in particular order - you need to remove some numbers first before adding them - imagine you are at limit and want to change 0->9. You can't type `9` and press del/backspace, but need to do it in the other way).

We should display an error if the input is not correct and let the user correct it..

Reply:
Note that we do support typing `1k` now (which translates to 1000 tokens) and this PR disables that. While the intention to fix this is noble, we should do it properly.

Reply:
> Note that we do support typing `1k` now (which translates to 1000 tokens) and this PR disables that. While the intention to fix this is noble, we should give it a little bit more love.

Oh we need to discuss this :D